storage/redis/peer_store.go

```diff
@@ -815,9 +815,8 @@ func (ps *peerStore) Stop() stop.Result {
 	go func() {
 		close(ps.closed)
 		ps.wg.Wait()
-		// chihaya does not clear data in redis when exiting.
-		// chihaya keys have prefix `IPv{4,6}_`.
-		close(c)
+		log.Info("exiting. chihaya does not clear data in redis when exiting. chihaya keys have prefix 'IPv{4,6}_'.")
```

Comment:
I'm so sorry to bother you again with this. Please add the `storage: ` prefix to the logged message so people know where it's coming from :)